bin/configs/java-helidon-server-mp_3_format_test.yaml

```diff
@@ -0,0 +1,10 @@
+generatorName: java-helidon-server
+library: mp
+outputDir: samples/server/others/java-helidon-server/v3/mp-format-test
```

Comment:
shall we add the new sample folders to the github workflow so that these will be tested moving forward?

Reply:
Yes, good catch. I'll push another commit right away.